Functions.Templates/Templates-v2/EventHubTrigger-Python/template.json

```diff
@@ -0,0 +1,204 @@
+{
+    "author": "Gavin Aguiar",
+    "name": "EventHub trigger",
+    "description": "$EventHubTrigger_description",
+    "programmingModel": "v2",
+    "language": "python",
+    "jobs": [
+        {
+            "name": "Create New Project with EventHubTrigger function",
+            "input" : {
+                "userCommand" : "createNewApp",
+                "assignTo" : "$(APP_FILENAME)"
+            },
+            "actions": [
+                "getFunctionName",
+                "getEventHubName",
+                "getConnnectionString",
+                "readFileContent_FunctionApp",
+                "replaceTokensInText_FunctionApp",
+                "writeFile_FunctionApp",
+                "showMarkdownPreview"
+            ]
+        },
+        {
+            "name": "Add EventHubTrigger function to the main file",
+            "actions": [
+                "getFunctionName",
+                "getEventHubName",
+                "getConnnectionString",
+                "readFileContent_FunctionBody",
+                "replaceTokensInText_FunctionBody",
+                "appendFileContent_FunctionApp"
+            ]
+        },
+        {
+            "name": "Create New Blueprint file",
+            "input" : {
+                "userCommand" : "createBluePrint",
+                "assignTo" : "$(BLUEPRINT_FILENAME)"
+            },
+            "actions": [
+                "getBluePrintFileName",
+                "getFunctionName",
+                "getEventHubName",
+                "getConnnectionString",
+                "readFileContent_BlueprintFile",
+                "replaceTokensInText_BlueprintFile",
+                "readFileContent_BlueprintBody",
+                "replaceTokensInText_BlueprintBody",
+                "appendFiles_Blueprint_BlueprintBody",

```

Comment:
At this point in this job's execution, the tokens that exist are `$(BLUEPRINT_FILENAME)`, `$(FUNCTION_NAME_INPUT)`, `$(EVENTHUB_NAME)`, `$(EVENTHUB_CONNECTION_STRING)`, `$(BLUEPRINT_FILE)` and `$(BLUEPRINT_BODY)`. 

`appendFiles_Blueprint_BlueprintBody` is defined like this:

```json
"name": "appendFiles_Blueprint_BlueprintBody",
"type": "AppendFiles",
"source": "$(BLUEPRINT)",
"assignTo" : "$(BLUEPRINT)"
```

Tooling would error here because the token `$(BLUEPRINT)` would be undefined or null.

Did you mean to append the content in `$(BLUEPRINT_BODY)` to the content in `$(BLUEPRINT_FILE)`?